packages/record/src/components/CopyToClipboardButton/index.js

```diff
@@ -26,7 +25,9 @@ export default function CopyToClipboardButton(props) {
     const tooltipRef = useRef();
     const [isVisible, setVisible] = useState(false);
     const [isCopied, setCopied] = useState(false);
-    const text = isCopied ? copiedText : copyText;
+    const copyProps = { id: 'click_to_copy', defaultMessage: 'Click to copy' };
+    const copiedProps = { id: 'copied', defaultMessage: 'Copied' };
```

Comment:
look at other components like `packages/payment/src/components/CreditCardPicker/messages.js`
where we define a messages file and then set an id following a format to and then document these ids in order to let the user use them to translate with their custom dictionarie